openstack-blueprint/input.yaml.tmpl

```diff
@@ -0,0 +1,30 @@
+# OPENSTACK SETTINGS
```

Comment:
maybe the name of the file is input.yaml, and accidentally it its input.yaml.tmpl?

Reply:
@dradX